README.md

```diff
@@ -4,7 +4,9 @@ The Visier Python Connector allows Python developers to query against the Visier
 
 ## Prerequisites
 This connector acts as a bridge between your Python application and Visier's cloud-hosted service infrastructure. To successfully connect to your Visier data, you need:
-* The URL domain name prefix. For example: `https://{vanity-name}.api.visier.io`.
+* The URL domain name prefix. For example: `https://{vanity-name}.api.visier.io`. To find your vanity name:
```

Comment:
Odd that it says 'For example`, when no example is given. Instead, we give a pattern. I'd like to take this opportunity to address this weirdness.

Reply:
Such as replacing {vanity-name} with jupiter? We can, but then the user won't know what we mean by "vanity name" in the subsequent instructions. A bigger change like:
> Your Visier vanity name and API URL. For example: `https://jupiter.api.visier.io`. To find your vanity name...